config/jobs/kubernetes/sig-scalability/sig-scalability-golang.yaml

```diff
@@ -25,7 +24,6 @@ periodics:
     path_alias: k8s.io/perf-tests
```

Comment:
does this repo's makefile handle clobbering GOTOOLCHAIN? if not we should add that somewhere

Reply:
I don't think so, but let's patch it there https://github.com/kubernetes/perf-tests/blob/master/golang/

Reply:
https://github.com/kubernetes/perf-tests/issues/3146